app/assets/templates/poll_tpl.jst.hbs

```diff
@@ -13,13 +13,26 @@
       {{#poll.poll_answers}}
         <div class="result-row">
           <input type="radio" name="vote" value="{{id}}"/>
-          <label>{{answer}}</label>
+            {{#if isCurrentUserVote}}
+              <label>
+                {{answer}}
+                  {{#linkToAuthor currentUser}}
+                    {{{personImage this 'small' 'micro'}}}
+                  {{/linkToAuthor}}
```

Comment:
I'm not sure if we should add a link to our own profile or just a simple avatar without the link.